spec/features/items/index_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+require 'rails_helper'
+
+RSpec.describe "Items Index", type: :feature do
+  describe "As a visitor, when I visit the Items Index page" do
+    it "I can see a list of items with their name, price, supermarket name, and count of customers who bought that item" do
+      @supermarket_1 = Supermarket.create(name: "Reasors", location: "Tulsa, OK")
+      @supermarket_2 = Supermarket.create(name: "Whole Foods", location: "Tulsa, OK")
+      @customer_1 = Customer.create(name: "Will")
+      @customer_2 = Customer.create(name: "Dee")
+      @item_1 = @supermarket_1.items.create(name: "Banana", price: 1)
+      @item_2 = @supermarket_1.items.create(name: "Apple", price: 1)
+      @item_3 = @supermarket_2.items.create(name: "Cilantro", price: 1)
+      @item_4 = @supermarket_2.items.create(name: "Eggs", price: 2)
+      @customer_item_1 = CustomerItem.create(customer_id: @customer_1.id, item_id: @item_1.id)
+      @customer_item_2 = CustomerItem.create(customer_id: @customer_1.id, item_id: @item_2.id)
+      @customer_item_3 = CustomerItem.create(customer_id: @customer_2.id, item_id: @item_3.id)
+      @customer_item_4 = CustomerItem.create(customer_id: @customer_2.id, item_id: @item_4.id)
+
+      visit '/items'
+
+      expect(page).to have_content(@item_1.name)
```

Comment:
Same idea here -- these tests would be more robust if you anchored them to the text next to them or used `within` blocks. On line 24, you're just testing that the number `1` appears somewhere on the page. But there are other items with the same price that could give you a false negative. There also might be a `1` on the page somewhere totally different (i.e. a "contact us" section with a phone number that contains a 1)

Reply:
Testing is a big part of this project! This week is an excellent time to dig into Capybara a bit more. Your testing is good! These are just some ways to make it more meaningful